### PR TITLE
Added the `!path` tag.

### LINKED ISSUE #1316: Command to teach how to add to Path
A commonly asked question I see is something where someone has to add a directory to the Path environment variable, and I think it would be nice to add some command that shows some links or tutorials on how to add something to Path for Windows and Unix-Based operating systems. This way, it will be faster and easier to direct someone to the correct resources and help them out, rather than trying to talk them through it.

### PR DESCRIPTION
**Pull request by Kronifer**
Closes #1316 

Added a short tag on what the PATH is, uses for it and resources on how to modify it.

### CODE REVIEW

**Comment by Xithrius** [comment]
Was this approved by a core developer?

File: bot/resources/tags/path.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+The PATH is a special environment variable that holds mulitple directory paths. The files in these directories can be executed from anywhere in the command line, on any directory.
+
+**Why would I want to modify the PATH?**
+
+There are multiple reasons you would want to modify your PATH. One reason is to use applications installed in other directories without including the path to execute the application. Another reason would be to make scripts more easily accessible.
+
+**Resources**
+
+• [What the heck is my PATH, and how do I get it to do what I want?](https://astrobiomike.github.io/unix/modifying_your_path)
+•  [How do I set or change the PATH system variable?](https://java.com/en/download/help/path.html)
```
Comment (ChrisLovering) [inline bot/resources/tags/path.md:12]: What's the reason for the extra space at the beginning here?

Comment (Xithrius) [reply bot/resources/tags/path.md:12]: @Kronifer 
```suggestion
• [How do I set or change the PATH system variable?](https://java.com/en/download/help/path.html)
```

**Comment by Xithrius** [comment]
@Kronifer are you planning on finishing this PR?

**Comment by Kronifer** [comment]
No, as im not sure this was explicitly approved.

**Pull request closed without merging**